rpc/block_modify.c

```diff
@@ -609,17 +610,14 @@ block_modify_cli_1_svc_st(blockModifyCli *blk, struct svc_req *rqstp)
     }
   }
 
-  errCode = 0;
```

Comment:
There is no need to set the errCode = -1 before GB_METALOCK_OR_GOTO macro:

```
231 # define  GB_METALOCK_OR_GOTO(lkfd, volume, errCode, errMsg, label)  \
232           do {                                                       \
233             struct flock _lock_ = {0, };                             \
234             _lock_.l_type = F_WRLCK;                                 \
235             if (glfs_posix_lock (lkfd, F_SETLKW, &_lock_)) {         \                                                                                     
236               LOG("mgmt", GB_LOG_ERROR, "glfs_posix_lock() on "      \
237                   "volume %s failed[%s]", volume, strerror(errno));  \
238               errCode = errno;                                       \
239               if (!errMsg) {                                         \
240                     GB_ASPRINTF (&errMsg, "Not able to acquire "     \
241                         "lock on %s[%s]", volume, strerror(errCode));\
242               }                                                      \
243               goto label;                                            \
244             }                                                        \
245           } while (0)          

```
In Line#238 it will set it to "errno" before goto the "label".